src/core/crypto.c

```diff
@@ -1551,7 +1551,13 @@ QuicCryptoProcessTlsCompletion(
         }
         Connection->Stats.ResumptionSucceeded = Crypto->TlsState.SessionResumed;
 
-        QuicSendSetSendFlag(&Connection->Send, QUIC_CONN_SEND_FLAG_PMTUD);
+        //
+        // A handshake complete means the peer has been validated. Trigger MTU
+        // discovery on path.
+        //
+        CXPLAT_DBG_ASSERT(Connection->PathsCount == 1);
```

Comment:
Any reason not to just assert `PathsCount >= 1`?

Reply:
Is it possible to get to this point with more then 1 path? I worry that if this assert fails then some paths might not get updates. The other option is to just trigger updates on all paths. That'd probably be better actually.

Reply:
Actually triggering on all paths in not safe. So we get in a weird state if we ever have more than 1 path here.

Reply:
I'm not saying we trigger for all paths, just that we assert that it's not 0.

Reply:
That doesn't work, because if somehow by this point a path that was 1 is now the active path, it might already have been triggered. Without that assert, its possible to double trigger MTU discovery. So to handle that, we'd need extra code to ensure that can't happen.